src/content/peerconnection/audio/js/main.js

```diff
@@ -82,14 +82,13 @@ function call() {
   pc2.onicecandidate = iceCallback2;
   pc2.onaddstream = gotRemoteStream;
   trace('Requesting local stream');
-  // Call into getUserMedia via the polyfill (adapter.js).
-  getUserMedia({
-      audio: true,
-      video: false
-    },
-    gotStream, function(e) {
-      alert('getUserMedia() error: ' + e.name);
-    });
+  navigator.mediaDevices.getUserMedia({
+    audio: true,
+    video: false
+  }).then(gotStream)
```

Comment:
Cannot we have `.then` on a new line like in https://github.com/webrtc/samples/pull/548/files#diff-e28becd7475fb7e59b9088adaa6eebacR60 ?